oauth2_http/javatests/com/google/auth/TestUtils.java

```diff
@@ -9,9 +9,12 @@
 import com.google.common.base.Splitter;
 import com.google.common.collect.Lists;
 
+import java.lang.RuntimeException;
```

Comment:
These imports look strange. You didn't change this file otherwise and you imported something from java.lang.